src/+dryair/calculateDryAirParameters.m

```diff
@@ -51,6 +57,9 @@
         AirVariabes.DhDZ(i) = DhDZ;
         AirVariabes.DTDZ(i) = DTDZ;
         AirVariabes.QL(i) = QL;
+        AirVariabes.QL_h(i) = QL_h(i); % added by Mostafa
+        AirVariabes.QL_T(i) = QL_T(i); % added by Mostafa
+        AirVariabes.QL_a(i) = QL_a(i); % added by Mostafa
```

Comment:
```suggestion
        % added by Mostafa
        AirVariabes.QL_h(i) = QL_h(i);
        AirVariabes.QL_T(i) = QL_T(i);
        AirVariabes.QL_a(i) = QL_a(i);
```

Reply:
Here, in front of each variable, can you please add the standard name of the variable?